utils.test.js

```diff
@@ -3,11 +3,79 @@ const {
   findImages,
   checkUrl,
   replaceConfluenceContent,
+  getArtifactIDFromPom,
 } = require('./utils.js');
 
 const fs = require('fs');
 
 describe('utils', function() {
+  it('getArtifactIDFromPom', async () => {
+    const content = '<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">\n' +
```

Comment:
Multi line string?

Reply:
I just copy and pasted the output of console.log

If you want to add an eslint rule, and thus support a --fix im for it. Otherwise meh